src/com/teamcautionrobotics/autonomous/commands2018/SetLiftCommand.java

```diff
@@ -0,0 +1,29 @@
+package com.teamcautionrobotics.autonomous.commands2018;
+
+import com.teamcautionrobotics.autonomous.Command;
+import com.teamcautionrobotics.robot2018.Lift;
+import com.teamcautionrobotics.robot2018.Lift.LiftLevel;
+
+public class SetLiftCommand implements Command {
+    
+    private Lift lift;
+    private LiftLevel liftLevel;
+
+    public SetLiftCommand(Lift lift, LiftLevel liftLevel) {
```

Comment:
Add an option to have SetLiftCommand wait until the lift is at the destination before finishing.